spec/unit/format_routing_constraint_spec.rb

```diff
@@ -1,56 +1,58 @@
-require "test_helper"
+RSpec.describe FormatRoutingConstraint, type: :model do
+  include ContentStoreHelpers
 
-class FormatRoutingConstraintTest < ActiveSupport::TestCase
   context "#matches?" do
```

Comment:
Should the first `context` be a `describe`

Reply:
Done!